src/jvmTest/java/org/snakeyaml/engine/usecases/references/ReferencesTest.java

```diff
@@ -97,17 +97,17 @@ public void referencesWithRecursiveKeysNotAllowedByDefault() {
   @Test
   @DisplayName("Parsing with aliases may take a lot of time, CPU and memory")
   public void parseManyAliasesForCollections() {
-    final var minDuration = Duration.ofMillis(400);
-    final var maxDuration = Duration.ofSeconds(15);
-    final var timeElapsed = assertTimeout(maxDuration, () -> {
+    final Duration minDuration = Duration.ofMillis(400);
```

Comment:
Setting the target version to 1.8 forced me to make the tests use 1.8-compatible constructs. I'm almost sure there's a better way to do it, like allowing higher version just for tests, but since I don't have much time before my vacation, I decided to use explicit typing for these several `var`s. I guess we don't have to rush it that much, so I can revisit once I'm back, also open to suggestions.

Reply:
I think you should be able to configure a higher compatibility version for tests by configuring the compilation tasks

```kotlin
tasks.named<JavaCompile>("compileTestJava") {
    sourceCompatibility = "11"
    targetCompatibility = "11"
}

tasks.named<KotlinCompile>("compileTestKotlinJvm") {
    compilerOptions {
        jvmTarget.set(JvmTarget.JVM_11)
    }
}
```

Reply:
We could update the tests to use Java 11, but we need to be able to test that the code works with Java 8 :)

`var` just isn't available in Java 8, so there's no alternative. Apart from converting the tests to Kotlin!